src/transformers/models/bloom/tokenization_bloom.py

```diff
@@ -0,0 +1,310 @@
+# coding=utf-8
```

Comment:
I thought we said no slow tokenizer?

Reply:
yes ! fixed in e580cb5a5e848b27c40eb835bc93159b0dbfe182